styles/_pages.scss

```diff
@@ -579,12 +617,21 @@
   }
 }
 
+.autoscroll-content {
+  animation: autoscroll 15s linear infinite;
+  white-space: nowrap;
+  overflow: hidden;
+  max-width: 300px;
+  /* adjust this value based on your design requirements */
```

Comment:
Did you copy-paste this, comment included, from somewhere else? The comment doesn't make sense here.